### ISSUE TITLE
Android windowSoftInputMode isn't responding for webview

### ISSUE THREAD

**Issue by jibon57** [NativeScript/NativeScript#5644]
Dear friend, we, the rest of the NativeScript community really
appreciate your feedback! While we are doing all we can to take care of every
issue, sometimes we get overwhelmed. Because of that, we will consider issues
that are not constructive or problems that cannot be reproduced "dead".
Additionally, we will treat feature requests or bug reports with unanswered
questions regarding the behavior/reproduction for more than 20 days "dead". All
"dead" issues will get closed.

### Did you verify this is a real problem by searching the [NativeScript Forum](http://forum.nativescript.org) and the [other open issues in this repo](https://github.com/NativeScript/nativescript/issues)?
Yes

### Tell us about the problem
Please, ensure your title is less than 63 characters long and starts with a capital
letter.

### Which platform(s) does your issue occur on?
Android

### Please provide the following version numbers that your issue occurs with:
- CLI: (run `tns --version` to fetch it) 3.4.3
- Cross-platform modules: (check the 'version' attribute in the
`node_modules/tns-core-modules/package.json` file in your project) 3.4.1
- Runtime(s): (look for the `"tns-android"` and `"tns-ios"` properties in the
`package.json` file of your project) 3.4.2
- Plugin(s): (look for the version number in the `package.json` file of your
project)

I am trying to view a website "http://m.longwellenglish.com/" using webview in home page of my app but keyboard is overlapping the contents when I am trying to type anything.

` <WebView id="webview" src="http://m.longwellenglish.com/"></WebView>`

I tried to change value of `windowSoftInputMode` in different way `adjustPan` or `adjustResize` or `adjustPan|adjustResize`. But none was helpful. Any suggestions? Thanks in advance. 

![image](https://user-images.githubusercontent.com/6080277/38549828-c4aefd0a-3cd6-11e8-9177-1ae5974474ea.png)

![image](https://user-images.githubusercontent.com/6080277/38549554-1b0650d2-3cd6-11e8-84f9-94e6eacd23a0.png)

![image](https://user-images.githubusercontent.com/6080277/38549584-300bba08-3cd6-11e8-8525-0cfd97c98c71.png)

**Comment by NickIliev** [comment]
@jibon57 it is expected that the property won't work for changes triggered in the WebView content.
The property is meant to work with the native iOS and Android controls and not with code executed inside a `WebView`. 
To make this one work with actions inside a web content you would need some additional custom logic and [nativescript-webview-interface](https://www.npmjs.com/package/nativescript-webview-interface) plugin. With this plugin, you can emit events from your WebView source. 

For example, you could emit an action when the user taps on editable text filed inside the web view and when you receive the event you could manually trigger the keyboard to pop up.

**Comment by jibon57** [comment]
@NickIliev Thanks for reply. I ain't sure if I was able to explain my problem clearly or not. In webview keyboard was working. It wasn't problem about keyboard but problem was content overleaping. When the keyboard was showing up that time the text field wasn't show up. I mean keyboard was overleaping that area. From google I learned to change `windowSoftInputMode` value but that wasn't working for my case.
